ipaserver/install/krainstance.py

```diff
@@ -150,6 +150,7 @@ def __spawn_instance(self):
         os.chown(cfg_file, pent.pw_uid, pent.pw_gid)
         self.tmp_agent_db = tempfile.mkdtemp(
                 prefix="tmp-", dir=paths.VAR_LIB_IPA)
+        self.tmp_agent_pwd = ipautil.ipa_generate_password()
```

Comment:
We are not using the password outside the scope of this method so please reduce this scope as well (remove `self.`)